src/main/resources/application-dev.yml

```
@@ -62,4 +62,9 @@ logging:
     org.springframework.security: DEBUG
     org.springframework.security.oauth2: DEBUG
     org.springframework.web.client.RestTemplate: DEBUG
-    com.example.dnd_13th_9_be: DEBUG
\ No newline at end of file
+    com.example.dnd_13th_9_be: DEBUG
+
+server:
+  tomcat:
+    max-http-form-post-size: 100MB
+    max-swallow-size: -1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)